protostar/commands/test/testing_live_logger.py

```diff
@@ -14,11 +14,23 @@
 
 class TestingLiveLogger:
     def __init__(
-        self, logger: Logger, testing_summary: TestingSummary, no_progress_bar: bool
+        self,
+        logger: Logger,
+        testing_summary: TestingSummary,
+        no_progress_bar: bool,
+        exit_first: bool,
     ) -> None:
         self._logger = logger
         self._no_progress_bar = no_progress_bar
         self.testing_summary = testing_summary
+        self.exit_first = exit_first
+
+    def exit_before_log(self, test_collector_result: "TestCollector.Result") -> None:
+        self.testing_summary.log(
+            logger=self._logger,
+            collected_test_cases_count=test_collector_result.test_cases_count,
+            collected_test_suites_count=len(test_collector_result.test_suites),
+        )
```

Comment:
```suggestion
    def log_testing_summary(self, test_collector_result: "TestCollector.Result") -> None:
        self.testing_summary.log(
            logger=self._logger,
            collected_test_cases_count=test_collector_result.test_cases_count,
            collected_test_suites_count=len(test_collector_result.test_suites),
        )
```

The name of this function is misleading. Now you can reuse this function in line 76.